assets/js/theme/global/menu.js

```diff
@@ -75,6 +82,27 @@ class Menu {
     onDocumentClick() {
         this.collapseAll()
     }
+
+    updateBodyMargin() {
+        if (this.$header.classList.contains('c-header__fixed')) {
```

Comment:
how about renaming `c-header__fixed` to `is-fixed`? I'd avoid binding js to a components' classes :)

Reply:
This was actually some code that was not needed anymore so I went ahead and deleted it :)